### ISSUE TITLE
how do we handle Alaska and Hawaii?

### ISSUE THREAD

**Issue by mapsam** [codeforamerica/fplapi#3]
They have different levels. So we should probably make an extra dictionary with their incomes. i.e. `fpl_ak` and `fpl_hi`

**Comment by daguar** [comment]
I might suggest allowing a query parameter of `state` with the two-letter abbreviation, and only respond with something different than the default if that argument is `HI` or `AK`.

eg

```
/api?state=AK
```

**Comment by mapsam** [comment]
Good call, Dave. It would be cool to just add this to the differences to the current dictionary for each state like:

``` python
"2015":
  "base": 11770,
  "rate": 4160,
  "ak": 23,
  "hi": 17
},
```

Then you could do the calculations accordingly.
